### PR TITLE
feature/menu

### PR DESCRIPTION
**Pull request by maxence-lefebvre**

### CODE REVIEW

File: src/main/preload.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// eslint does not allow me to put electron in devDependencies
+// electron-builder does not allow me to put electron in dependencies
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { ipcRenderer } = require('electron');
+const { EventEmitter } = require('events');
+
+window.electron = {
```
Comment (gautier-lefebvre) [inline src/main/preload.js:7]: isn't it `global` on the main process ?

File: src/renderer/services/ipc/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { uniqueId } from 'lodash';
+
+const { ipcRenderer } = window && window.electron;
```
Comment (gautier-lefebvre) [inline src/renderer/services/ipc/index.js:3]: Is there a reason why don't you put the code in `common` ?

Comment (maxence-lefebvre) [reply src/renderer/services/ipc/index.js:3]: which code ?

File: src/main/preload.js (hunk 1 above)
Comment (maxence-lefebvre) [reply src/main/preload.js:7]: > preload String (optional) - Specifies a script that will be loaded before other scripts run in the page. This script will always have access to node APIs no matter whether node integration is turned on or off. The value should be the absolute file path to the script. When node integration is turned off, the preload script can reintroduce Node global symbols back to the global scope. See example here.

https://electronjs.org/docs/api/browser-window

**Review by gautier-lefebvre — APPROVED** [review]

**Pull request merged**